### PR TITLE
TestCommand for ConfigStore

### PR DESCRIPTION
**Pull request by RonConfigu**

### CODE REVIEW

File: ts/packages/cli/src/commands/test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Flags } from '@oclif/core';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```
Comment (RonConfigu) [inline ts/packages/cli/src/commands/test.ts:3]: ignore this.
this is just until the SDK will update

File: ts/packages/cli/src/commands/test.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    }),
+  };
+
+  public async run(): Promise<void> {
+    const store = await this.getStoreInstanceByStoreFlag(this.flags.store);
+    const { clean } = this.flags;
+    const testResult = await new TestCommand({ store, clean }).run();
+    // TODO: better messages from @ran
+    if (testResult) this.log(`Credentials and write access available`);
+    else this.log(`Please login to Configu using: configu store upsert --type "configu"`);
```
Comment (RonConfigu) [inline ts/packages/cli/src/commands/test.ts:29]: @rannn505 please refactor messages.

File: ts/packages/ts/src/commands/TestCommand.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { Command } from '../Command';
+import { ConfigStore } from '../ConfigStore';
+
+export type TestCommandParameters = {
+  store: ConfigStore;
+  clean?: boolean;
+};
+
+export class TestCommand extends Command<boolean> {
```
Comment (rannn505) [inline ts/packages/ts/src/commands/TestCommand.ts:9]: see previous comment regarding void and error.

File: ts/packages/ts/src/commands/TestCommand.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    super(parameters);
+  }
+
+  async run(): Promise<boolean> {
+    const testConfig = {
+      set: '',
+      key: 'CONFIGU_TEST',
+      value: Date.now().toString(),
+    };
+    try {
```
Comment (rannn505) [inline ts/packages/ts/src/commands/TestCommand.ts:20]: see previous comment regarding error, dump try catch and let the store.set() to throw the "real" error.

**Review by rannn505 — CHANGES_REQUESTED** [review]
@RonConfigu Smooth implementation, nicely done 🚀 . see minor comment before approve

File: ts/packages/cli/src/commands/test.ts (hunk 2 above)
Comment (RonConfigu) [reply ts/packages/cli/src/commands/test.ts]: done

File: ts/packages/ts/src/commands/TestCommand.ts (hunk 3 above)
Comment (RonConfigu) [reply ts/packages/ts/src/commands/TestCommand.ts]: done

File: ts/packages/ts/src/commands/TestCommand.ts (hunk 4 above)
Comment (RonConfigu) [reply ts/packages/ts/src/commands/TestCommand.ts]: done

**Review by rannn505 — APPROVED** [review]
@RonConfigu Nice work :)

**Pull request merged**